### PR TITLE
users: Fix "user is locked" detection; pkg, examples: Avoid assigning .innerHTML

### PR DESCRIPTION
**Pull request by martinpitt**
These were some findings from evaluating https://semgrep.dev

### CODE REVIEW

File: pkg/playground/speed.js [hunk 1]
```diff
@@ -28,11 +28,11 @@ function update() {
...
         proc.then(function(data) {
             const parts = data.split("\n");
-            pid.innerHTML = parts[0];
-            memory.innerHTML = parts[1];
+            pid.textContent = parts[0];
+            memory.textContent = parts[1];
             proc = null;
         }, function(ex) {
-            memory.innerHTML = String(ex);
+            memory.textContent = String(ex);
```
Comment (cockpituous) [inline pkg/playground/speed.js:35]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19047-20230630-121653-41e6f122-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/playground/speed.js [hunk 2]
```diff
@@ -200,7 +200,7 @@ function download(ev) {
 function spawn() {
     stop();
 
-    document.getElementById("spawn-result").innerHTML = "Running...";
+    document.getElementById("spawn-result").textContent = "Running...";
```
Comment (cockpituous) [inline pkg/playground/speed.js:203]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19047-20230630-121653-41e6f122-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/playground/speed.js [hunk 3]
```diff
@@ -212,17 +212,17 @@ function spawn() {
     channel.stream(data => {
         console.log("spawn: stream block length", data.length);
         total += data.length;
-        document.getElementById("spawn-output").innerHTML = data;
+        document.getElementById("spawn-output").textContent = data;
```
Comment (cockpituous) [inline pkg/playground/speed.js:215]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19047-20230630-121653-41e6f122-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/playground/speed.js [hunk 4]
```diff
@@ -212,17 +212,17 @@ function spawn() {
...
     });
     channel.then(() => {
         console.log("spawn: command finished successfully");
         stop();
-        document.getElementById("spawn-result").innerHTML = "success";
+        document.getElementById("spawn-result").textContent = "success";
```
Comment (cockpituous) [inline pkg/playground/speed.js:220]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19047-20230630-121653-41e6f122-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/playground/speed.js [hunk 5]
```diff
@@ -212,17 +212,17 @@ function spawn() {
...
     });
     channel.catch(ex => {
         console.log("spawn: command failed", JSON.stringify(ex));
         stop();
-        document.getElementById("spawn-result").innerHTML = `failed with exit code ${ex.exit_status}: ${ex.message}`;
+        document.getElementById("spawn-result").textContent = `failed with exit code ${ex.exit_status}: ${ex.message}`;
```
Comment (cockpituous) [inline pkg/playground/speed.js:225]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19047-20230630-121653-41e6f122-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by jelly — APPROVED** [review]
:robot: :heavy_check_mark:

**Pull request merged**